pkgs/by-name/bi/bitwarden-cli/package.nix

```diff
@@ -34,6 +34,7 @@ buildNpmPackage rec {
   ] ++ lib.optionals stdenv.isDarwin [
     cctools
     xcbuild.xcrun
+    xcbuild.xcodebuild
```

Comment:
```suggestion
    xcodebuild
```

Reply:
> error: attribute 'xcodebuild' missing

       at /ofborg/checkout/1/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-0/pkgs/by-name/bi/bitwarden-cli/package.nix:37:5:

           36|     xcbuild.xcrun
           37|     xcbuild.xcodebuild
             |     ^
           38|   ];

Reply:
@khaneliman With the suggested change there is an error:
```
error: undefined variable 'xcodebuild'
       at /home/runner/work/nixpkgs/nixpkgs/pkgs/by-name/bi/bitwarden-cli/package.nix:38:5:
           37|     xcbuild.xcrun
           38|     xcodebuild
             |     ^
           39|   ];
```

Reply:
Would still needed to be added to inputs.